manifests/init.pp

```diff
@@ -101,6 +101,25 @@
 # [*token_retransmits_before_loss_const*]
 #   How many token retransmits before forming a new configuration
 #
+# [*join*]
+#   This timeout specifies in milliseconds how long to wait for join messages
+#   in the membership protocol.
+#   Default to 50
+#
+# [*consensus*]
+#   This timeout specifies in milliseconds how long to wait for consensus to be
+#   achieved before starting a new round of membership configuration.
+#   The minimum value for consensus must be 1.2 * token. This value will be
+#   automatically calculated at 1.2 * token if the user doesn't specify a
+#   consensus value.
+#   Defaults to false,
+#
+#[*max_messages*]
+#   This constant specifies the maximum number of messages that may be sent by
+#   one processor on receipt of the token. The max_messages parameter is limited
+#   to 256000 / netmtu to prevent overflow of the kernel transmit buffers.
+#   Defaults to 17
```

Comment:
Why was it set to 20 ?
By default in the man corosync.conf it's set to 17 messages. That's why I just fit to the original values.